reflex/.templates/apps/basetemplate/code/sidebar.py

```diff
@@ -0,0 +1,125 @@
+import reflex as rx
+
+from .state import State
+from .styles import *
+
+
+def sidebar_item(text: str, icon: str, url: str) -> rx.Component:
+    return rx.hstack(
```

Comment:
I think this should return an `rx.link` with a real `href` and with the `rx.hstack` as a child. As it is now, when I actually create a new page and update the sidebar, clicking the sidebar_item does nothing except update the highlight.